tools/test/collect_coverage.sh

```diff
@@ -140,7 +140,7 @@ elif [[ "$COVERAGE_LEGACY_MODE" ]]; then
   #                         the current directory
   # -d "${COVERAGE_DIR}"  - Directory to search for .gcda files
   # -o "${COVERAGE_OUTPUT_FILE}" - Output file
-  /usr/bin/lcov -c --no-external --ignore-errors graph -q \
+  lcov -c --no-external --ignore-errors graph -q \
```

Comment:
To not break any previous usages you can replace lcov with $LCOV, where:

LCOV=$(which lcov)
if [[ ! -x $LCOV ]]; then
    LCOV=/usr/bin/lcov
fi

Reply:
Sure, makes sense.